spec/spec_helper_acceptance.rb

```diff
@@ -627,3 +411,28 @@ def puppet_resource_should_show(property_name, value=nil)
     expect(@result.stdout).to match(regex)
   end
 end
+
+def beaker_opts
+  azure_cert = is_windows? ? WINDOWS_DOS_FORMAT_AZURE_CERT : LINUX_AZURE_CERT
+  @env ||=
+  {
+    debug: true,
+    trace: true,
+    environment: {
+      'AZURE_CLIENT_ID' => ENV['AZURE_CLIENT_ID'],
+      'AZURE_CLIENT_SECRET' => ENV['AZURE_CLIENT_SECRET'],
+      'AZURE_MANAGEMENT_CERTIFICATE' => azure_cert,
+      'AZURE_SUBSCRIPTION_ID' => ENV['AZURE_SUBSCRIPTION_ID'],
+      'AZURE_TENANT_ID' => ENV['AZURE_TENANT_ID'],
+    }
+  }
+end
+
+def is_windows?(host = nil)
```

Comment:
This method isn't explicit about the third option, when host isn't set and default is undefined. It might be better formulated to have `elsif defined(default)` and then an else clause which either throws an error or returns false. At the moment the third option isn't explicit.